core/src/style/textStyleBuilder.cpp

```diff
@@ -537,11 +537,13 @@ TextStyle::Parameters TextStyleBuilder::applyRule(const DrawRule& _rule,
             }
         }
 
-
         _rule.get(StyleParamKey::text_transition_hide_time, p.labelOptions.hideTransition.time);
         _rule.get(StyleParamKey::text_transition_selected_time, p.labelOptions.selectTransition.time);
         _rule.get(StyleParamKey::text_transition_show_time, p.labelOptions.showTransition.time);
 
+        if (!_rule.get(StyleParamKey::text_buffer, p.labelOptions.buffer)) {
```

Comment:
It's unclear, have you tried and seen that there was no inheritance? I don't see any mention of that in https://github.com/tangrams/tangram/pull/306.